pkg/ingester/ingester_test.go

```diff
@@ -524,7 +525,7 @@ func BenchmarkIngesterPush(b *testing.B) {
 	limits := defaultLimitsTestConfig()
 
 	const (
-		series  = 100
+		series  = 10000
```

Comment:
This is testing a radically different mix of operations, and in particular very different from the default Prometheus config. I don't think it's justified.

Reply:
I don't think I understand what you mean here

Reply:
The setup/teardown costs are divided over 100,000 series instead of 100.

Put it another way: why did you make this change?

Reply:
I didn't think much before making the change to 10,000 series, I thought it would make the test more intensive. I have reverted back to 100 series anyway and have updated the benchmark results in the PR description

Here is the updated benchmark result:
```
benchmark                                        old ns/op     new ns/op     delta
BenchmarkIngesterPush/encoding=DoubleDelta-8     16431496      15754699      -4.12%
BenchmarkIngesterPush/encoding=Varbit-8          16342108      15671923      -4.10%
BenchmarkIngesterPush/encoding=Bigchunk-8        16409123      15624513      -4.78%

benchmark                                        old allocs     new allocs     delta
BenchmarkIngesterPush/encoding=DoubleDelta-8     32348          12139          -62.47%
BenchmarkIngesterPush/encoding=Varbit-8          32341          12138          -62.47%
BenchmarkIngesterPush/encoding=Bigchunk-8        32344          12137          -62.48%

benchmark                                        old bytes     new bytes     delta
BenchmarkIngesterPush/encoding=DoubleDelta-8     3324933       2768749       -16.73%
BenchmarkIngesterPush/encoding=Varbit-8          3341311       2764124       -17.27%
BenchmarkIngesterPush/encoding=Bigchunk-8        3338947       2769880       -17.04%
```

Reply:
Thanks! 

Go benchmarks are repeated enough times to make it run for more than 1 second, so you didn't make it more intensive overall.